batch-models/src/main/resources/schema.sql

```diff
@@ -0,0 +1,126 @@
+   data_last_generated_on VARCHAR(255)
+);
+
+-- Table: org_hierarchy
+CREATE TABLE org_hierarchy(
+   mdo_id VARCHAR(255) PRIMARY KEY NOT NULL,
+   mdo_name  VARCHAR(255),
+   is_cbp_provider VARCHAR(64),
+   ministry VARCHAR(255),
+   department VARCHAR(255),
+   organization VARCHAR(255),
+   data_last_generated_on DATE
+);
+
+-- Table: user_enrolment
+CREATE TABLE user_enrolment(
+   user_id VARCHAR(255) NOT NULL,
+   batch_id VARCHAR(255) NOT NULL,
+   cbp_id  VARCHAR(255)  NOT NULL,
+   cbp_progress_percentage FLOAT,
+   completed_on VARCHAR(255),
+   certificate_generated VARCHAR(255),
+   certificate_generated_on VARCHAR(255),
+   user_rating FLOAT,
+   resource_count_consumed INTEGER,
+   enrolled_on VARCHAR(255),
+   user_consumption_status VARCHAR(255),
+   data_last_generated_on VARCHAR(255),
+   PRIMARY KEY (user_id, cbp_id, batch_id)
+);
+
+-- Table: cbp
+CREATE TABLE cbp(
+   cbp_id VARCHAR(255) PRIMARY KEY NOT NULL,
+   cbp_provider_id VARCHAR(255),
+   cbp_provider_name VARCHAR(255),
+   cbp_name VARCHAR(255),
+   cbp_type VARCHAR(255),
+   batch_id VARCHAR(255),
+   batch_name VARCHAR(255),
+   batch_start_date VARCHAR(255),
+   batch_end_date VARCHAR(255),
+   cbp_duration VARCHAR(255),
+   cbp_rating FLOAT,
+   last_published_on VARCHAR(255),
+   cbp_retired_on VARCHAR(255),
+   cbp_status VARCHAR(255),
+   resource_count INTEGER,
+   total_certificates_issued INTEGER,
+   cbp_substatus VARCHAR(255),
+   data_last_generated_on VARCHAR(255)
+);
+
+-- Table: assessment_detail
+CREATE TABLE assessment_detail(
+   user_id VARCHAR(255)  NOT NULL,
+   cbp_id VARCHAR(255) NOT NULL,
+   assessment_id VARCHAR(255) NOT NULL,
+   assessment_name VARCHAR(255) ,
+   assessment_type VARCHAR(255),
+   assessment_duration VARCHAR(255),
+   time_spent_by_the_user VARCHAR(255),
+   completion_date VARCHAR(255),
+   score_achieved FLOAT,
+   overall_score FLOAT,
+   cut_off_percentage FLOAT,
+   total_question INTEGER,
+   number_of_incorrect_responses INTEGER,
+   number_of_retakes INTEGER,
+   data_last_generated_on VARCHAR(255),
+   PRIMARY KEY (assessment_id, cbp_id, user_id)
+);
+
+-- Table: bp_enrollments
+CREATE TABLE bp_enrollments(
+   user_id VARCHAR(255) NOT NULL,
+   cbp_id  VARCHAR(255)    NOT NULL,
+   batch_id VARCHAR(255) NOT NULL,
+   batch_location VARCHAR(255),
+   component_name VARCHAR(255),
+   component_id VARCHAR(255),
+   component_type VARCHAR(255),
+   component_mode VARCHAR(255),
+   component_status VARCHAR(255),
+   component_duration VARCHAR(255),
+   component_progress_percentage FLOAT,
+   offline_session_date VARCHAR(255),
+   offline_session_start_time VARCHAR(255),
+   offline_session_end_time VARCHAR(255),
+   offline_attendance_status VARCHAR(255),
+   instructors_name VARCHAR(255),
+   program_coordinator_name VARCHAR(255),
+   data_last_generated_on VARCHAR(255)
+);
+
+-- Create PostgreSQL user and grant privileges
+CREATE USER postgres WITH PASSWORD 'Password@12345678';
```

Comment:
@wilkysingh-tarento  We should not have credentials in the sql. This can be mentioned in the confluence page which is private.